### DISCUSSION TITLE
"Unblockable" Desync Kick

### DISCUSSION THREAD

**Discussion by GitCat3** [Mr-X-GTA/YimMenu#49] [General]
Is there any code I can look at for how the unblockable Desync Kick works just to see how it's done and what it exploits? Thanks!

**Comment by Mr-X-GTA** [comment]
No, all you can do is spoof your NAT type, which I have already implemented in YimMenu

**Reply by GitCat3** [reply]
Spoofing your nat type allows you to desync kick people without them being able to block it? How does that work if you don't mind me asking?

**Reply by Mr-X-GTA** [reply]
The NAT type is used to decide whether the complainer or the complainee is kicked

**Reply by GitCat3** [reply]
so if you change your nat type and then complain about someone you can force them to get kicked?
